src/app/zap-templates/zcl/data-model/chip/air-quality-cluster.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <name>Air Quality</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes for reporting air quality classification</description>
+    <code>0x005B</code>
+    <define>AIR_QUALITY_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="AIR_QUALITY" type="AirQualityEnum" min="0" max="5" writable="false" isNullable="false" default="0" optional="false">AirQuality</attribute>
```

Comment:
How about `SupportedLevels` ?

Reply:
I assume this is based on https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/6559; it does say that above.

That unfortunately means you have to use a spec from the PR to compare to.

Reply:
I would also loved to have this PR merged but it seems policy that during the ballot nothing gets merged 🙁

Reply:
Right, that was just context for @andy31415 so he knows what he should look for....